.buildkite/scripts/steps/serverless/build_and_deploy.sh

```diff
@@ -137,8 +137,35 @@ EOF
   buildkite-agent meta-data set pr_comment:early_comment_job_id "$BUILDKITE_JOB_ID"
 }
 
+# This is the integration with the observability-test-environments
+create_github_issue_oblt_test_environments() {
+
+cat <<EOF > .issue-body
```

Comment:
This is the contract with the automation, using a GitHub issue template.

Reply:
Tiny nit, mind if we follow the pattern at https://github.com/elastic/kibana/blob/b8c4dd24b3d035d9ed20d5429b909a1daa6eb6d7/.buildkite/scripts/steps/serverless/build_and_deploy.sh#L46?

Reply:
```suggestion
GITHUB_ISSUE=$(mktemp --suffix ".md") 
cat <<EOF > "$GITHUB_ISSUE"
```